### DISCUSSION TITLE
Starstruck makes no sense, starcount is wrong and same repo mentioned twice

### DISCUSSION THREAD

**Discussion by specialunderwear** [community/community#35324] [Other Feature Feedback, Questions, & Ideas] [answered]
<img width="344" alt="image" src="https://user-images.githubusercontent.com/52858/194321263-f727b146-e4bb-44aa-9fe2-f860b2d1520f.png">

The repo that is mentioned twice really has 1.6k stars so this makes no sense at all:

https://github.com/specialunderwear/Hosts.prefpane

**Comment by spenserblack** [comment, marked as answer]
The achievements show *milestones,* not current values. You reached >=16 stars in `django-easymode` (x1), >=128 stars in `Hosts.prefpane` (x2), and >=512 stars again in `Hosts.prefpane` (x3).
